loggercontext/logger_context.go

```diff
@@ -0,0 +1,48 @@
+// Package loggercontext stores and accesses the container struct in
+// context.Context.
+package loggercontext
+
+import (
+	"context"
+)
+
+// key is an unexported type for keys defined in this package. This prevents
+// collisions with keys defined in other packages.
+type key string
+
+// loggerKey is the key for logger struct values in context.Context. Clients use
+// loggercontext.NewContext and loggercontext.FromContext instead of using this
+// key directly.
+var loggerKey key = "logger"
+
+// Container is a communication structure used to transport information from one
+// resource to another. Container move between resource during reconciliation
+// within the dispatched context.
```

Comment:
> Container move between resource during reconciliation

I think micrologger is generic,  we shouldn't put in info about reconciliation of resources in the godoc.

Reply:
Bad copy/paste. I can fix this.

Reply:
See https://github.com/giantswarm/micrologger/pull/9.